proxy/http2/HPACK.cc

```diff
@@ -181,10 +183,10 @@ static const StaticTable STATIC_TABLE[] = {{"", ""},
 */
 static constexpr uint32_t HPACK_HDR_HEAP_THRESHOLD = sizeof(MIMEHdrImpl) + sizeof(MIMEFieldBlockImpl) * (2 + 7 + 15);
 
-/******************
- * Local functions
- ******************/
-static inline bool
+//
+// Local functions
+//
+bool
```

Comment:
Why are you changing this to have global scope?  Did you mean to put it into an anonymous namespace?